development/bin/ensure_release_notes.py

```diff
@@ -0,0 +1,89 @@
+"""Ensure that release notes exist for a given version."""
+
+import argparse
+import sys
+
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import tomli as tomllib
```

Comment:
I think the more pythonic way to do this is just
```suggestion
try:
    import tomllib
except ImportError:
    import tomli as tomllib
```